<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMVtool</class>
 <widget class="QMainWindow" name="AMVtool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>520</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>AMVtool (Beta 4)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>231</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertFiles">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFiles">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="configEncSettings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>581</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="configAll">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure All</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileExists">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>If Output File Exists:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="ifExists">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ask</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Overwrite</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Skip</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
